pinot-spi/src/main/java/org/apache/pinot/spi/utils/validator/PinotJsonSchemasValidator.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.pinot.spi.utils.validator;
+
+public class PinotJsonSchemasValidator extends BaseJsonValidator {
+
+  private static final String BASE_SCHEMA_DIR = "/schemas";
+
+  private static final String TABLE_CONFIG_SCHEMA = BASE_SCHEMA_DIR + "/tableConfig.json";
```

Comment:
Do we need 2 separate constants for this ?

Reply:
For now, we don't but, in case we want to add schema validation to another thing.
But I'll remove it anyway